File: lab_5/lab_5_1.pas
program zzz;
const k=10;
var
  sch, i: integer;
  sum:integer;
  a : array [1..k] of integer;
begin
  sch:=0;
  sum:=0;
  // writeln ('vvtdite k ');
  // read (k);
  for i:=1 to k do begin
    write ('a[', i, ']= ');
    read (a[i]);
    if a[i]<0 then
      sch:=sch+1
    else
      sum+=a[i];
  end;
  writeln ('kollichestvo otrecatelnih =', sch);
  writeln ('summa polojitelnih = ', sum);
  readln(i);
end.

File: lab_5/lab_5_1.cpp
#include <iostream>
#include <conio.h>
int main(int argc, char const *argv[]) {
  int k, sum, sch=0;
  std::cout << "vvtdite k" << '\n';
  std::cin >> k;
  double a[k];
  for (int i=0; i<k; i++)
    {
      std::cout << "a[" << i <<"]=";
      std::cin >> a[i];
      if (a[i]<0)
      sch++;
      else
      sum+=a[i];
    }
  std::cout << "kollichestvo otrecatelnih = " <<sch<< '\n';
  std::cout << "summa polojitelnih = " <<sum<< '\n';
  return 0;
}
